atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -97,14 +97,36 @@ def run_in_orchestrator(self):
         compute their replacements and set build arg for worker.
 
         Exclude CSVs which already have a relatedImages section.
+
+        Returns operator metadata in format
+        related_images:
+          pullspecs:  # list of all related_images_pullspecs
+            - original: <original-pullspec1>  # original pullspec in CSV file
+              new: <new pullspec>   # new pullspec computed by this plugin
+              pinned: <bool>  # plugin pinned digest
```

Comment:
Can you clarify the difference between `pinned` and `modified`?

Reply:
added